pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -51,40 +54,38 @@ import (
 
 // Files that may appear in a backup directory.
 const (
-	// backupManifestChecksumSuffix indicates where the checksum for the manifest
+	// BackupManifestChecksumSuffix indicates where the checksum for the manifest
 	// is stored if present. It can be found in the name of the backup manifest +
 	// this suffix.
-	backupManifestChecksumSuffix = "-CHECKSUM"
+	BackupManifestChecksumSuffix = "-CHECKSUM"
```

Comment:
[future,discussion] Some of these being public is a signal that perhaps we need some other methods in this package. That is, eventually do we expect it to be the case that most users of this package don't need to worry about the internal structure of these constants.